src/WebJobs.Extensions.OpenAI/Models/ChatMessage.cs

```diff
@@ -25,7 +27,7 @@ public ChatMessageEntity(string content, string role)
     public string Content { get; set; }
 
     /// <summary>
-    /// Gets or sets the role of the chat agent (“system”, “user”, "assistant", "function" or "tool").
+    /// Gets or sets the role of the chat agent.
     /// </summary>
     public string Role { get; set; }
```

Comment:
The problem I ran into when trying this was that `ChatRole` types don't serialize to JSON correctly. For example, it would serialize to `{}` instead of `"user"` or `"assistant"`. I decided to leave it as string so that serialization would work as expected.